pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -22,7 +23,18 @@ class Range(Module):
   is_original_torch_module = False
 
   def create_returnn_layer_dict(self, limit, start, delta, dtype, sparse=False) -> Dict[str, Any]:
-    return {"class": "range", "limit": limit, "start": start, "delta": delta, "dtype": dtype, "sparse": sparse}
+    if isinstance(limit, Tensor) and limit.dim_tag.dimension is None:
+      return {"class": "range_from_length", "from": self._get_input_layer_name(limit)}
+    else:
+      assert isinstance(limit, int)
+      return {"class": "range", "limit": limit, "start": start, "delta": delta, "dtype": dtype, "sparse": sparse}
+
+  def _get_output_shape_from_returnn(self,
+                                    inputs_flat: List, layer: LayerBase
+                                    ) -> Tuple[Tuple[int, ...], Dict[int, int]]:
+    torch_shape = ((int(inputs_flat[0]) - int(inputs_flat[1])) // inputs_flat[2],)
```

Comment:
And here, how to infer the integer values for the torch side if we get a `Tensor`.

Reply:
This here remains also after fixing `_make_tensor`

Reply:
Simply `inputs_flat[0].numpy()`?
Btw, you better should name the args:
```python
limit, start, delta, *_ = inputs_flat
```

Reply:
Ah then we need to fix the `_numpy_buffer` for this tensor in `SizeValue.as_tensor()`.

Reply:
I pushed this anyway now, just for reference. The test passes, but I think this is not correct yet. The output also seems wrong
```
layer root/'output' output: Data{'output_output', [T|'Range_input_len'[]], dtype='int32'}
```

Reply:
> Ah then we need to fix the _numpy_buffer for this tensor in SizeValue.as_tensor().

Ah sorry, this did not show up before. I'll fix this.

Reply:
Yes, now the `torch_shape` is correct.

But the output seems still incorrect, doesn't it? I mean why is it `T`?
```
layer root/'output' output: Data{'output_output', [T|'Range_input_len'[]], dtype='int32'}
```

Reply:
`T` is just the marker for the `time_dim_axis`. This should not be relevant here.

Reply:
Alright 👍